dependency-check-supressions.xml

```diff
@@ -83,4 +83,11 @@
         <packageUrl regex="true">^pkg:maven/org\.dom4j/dom4j@.*$</packageUrl>
         <cve>CVE-2020-10683</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+            file name: websocket-extensions:0.1.4
+   ]]></notes>
+        <packageUrl regex="true">^pkg:npm/websocket\-extensions@.*$</packageUrl>
+        <vulnerabilityName>CVE-2020-7663</vulnerabilityName>
```

Comment:
Why is this suppressed?

Reply:
the most recent version of websocket-extensions (0.1.4) has a vulnerability in it that has not yet been patched. Until a fix is released vulnerability needs to be suppressed for the build to pass.

Reply:
It should be noted that react-scripts is what bring in websocket-extensions.React-scripts is a devDependency, not a production dependency so this shouldn't expose production instances to increased vulnerabilities.